### PR TITLE
Users may now make inline edits to color names. 

### LINKED ISSUE #407: Colors Palette: Allow users to change color name by clicking on the Color name
image

For example, clicking on "Medium Purple" should show an input box that allows the user to modify the current name, instead of the user having to click on the pencil icon and then choose `Edit Color Name`

The user should be able to edit inline on the same page. Similar to how the UI of the HSL values boxes.

### PR DESCRIPTION
**Pull request by Bismay5467**
Resolves #407

### CODE REVIEW

File: packages/core/src/components/ColorPalette/ColorDisplay.tsx [hunk 1]
```diff
@@ -124,7 +120,28 @@ export function ColorDisplay({
         mb={5}
         style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}
       >
-        <Text style={{ fontWeight: 600, fontSize: '1.5rem' }}>{colorName}</Text>
+        <Input
```
Comment (Pranav2612000) [inline packages/core/src/components/ColorPalette/ColorDisplay.tsx:123]: I think the expected behaviour should be that the input box opens up only after double-clicking on the color name ( rather than single-click, which is the behaviour in this PR ). Thoughts @alexdanilowicz ?

File: packages/core/src/components/ColorPalette/ColorDisplay.tsx [hunk 2]
```diff
@@ -133,9 +150,6 @@ export function ColorDisplay({
             <MenuItem onClick={() => onBaseModalOpen()}>
               Edit Base Color
             </MenuItem>
-            <MenuItem onClick={() => onColorNameModalOpen()}>
-              Edit Color Name
-            </MenuItem>
```
Comment (Pranav2612000) [inline packages/core/src/components/ColorPalette/ColorDisplay.tsx:138]: I think we should keep this way of updating name as well.

File: packages/core/src/components/ColorPalette/ColorDisplay.tsx (hunk 1 above)
Comment (alexdanilowicz) [reply packages/core/src/components/ColorPalette/ColorDisplay.tsx:123]: Agreed @Pranav2612000.

File: packages/core/src/components/ColorPalette/ColorDisplay.tsx (hunk 2 above)
Comment (alexdanilowicz) [reply packages/core/src/components/ColorPalette/ColorDisplay.tsx:138]: Agreed, yes let's keep the name

File: packages/mirrorful/editor/src/blocks.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import '../.mirrorful/theme.css';
+/* @import '../.mirrorful/theme.css'; */
```
Comment (alexdanilowicz) [inline packages/mirrorful/editor/src/blocks.css:1]:
```suggestion
@import '../.mirrorful/theme.css';
```

@Bismay5467 why was this removed?

**Comment by alexdanilowicz** [comment]
Building it without the removal of theme.scss in blocks so I can ensure it still works and build fine. then will merge this in.
